Examples/UIExplorer/js/ListViewExample.js

```diff
@@ -62,9 +62,9 @@ var ListViewSimpleExample = React.createClass({
         noSpacer={true}
         noScroll={true}>
         <ListView
+          removeClippedSubviews={true}
           dataSource={this.state.dataSource}
           renderRow={this._renderRow}
-          renderScrollComponent={props => <RecyclerViewBackedScrollView {...props} />}
```

Comment:
Got a PR to get rid of it, https://github.com/facebook/react-native/pull/11999 if you want to ship it :)